### PR TITLE
Remove executor config variable from task env

### PR DESCRIPTION
**Pull request by janisz**
Fixes: #84

### CODE REVIEW

File: command.go [hunk 1]
```diff
@@ -136,7 +137,8 @@ func NewCommand(commandInfo mesos.CommandInfo, env []string, options ...func(*ex
 	//		similar to how POSIX exec families launch processes (i.e.,
 	//		execlp(value, arguments(0), arguments(1), ...)).
 	cmd := exec.Command("sh", "-c", commandInfo.GetValue()) // #nosec
-	cmd.Env = combineExecutorAndTaskEnv(env, commandInfo.GetEnvironment())
+	removeExecutorConfigurationEnv()
```
Comment (medzin) [inline command.go]: It is not safe to modify global state of the executor process. Let this function return the cleared slice (without modifying the process environment) and use it in the started command.

File: command_test.go [hunk 2]
```diff
@@ -23,6 +23,21 @@ func TestIfNewCancellableCommandReturnsCommand(t *testing.T) {
...
+	os.Setenv("TEST", "z")
+	os.Setenv("_ALLEGRO_EXECUTOR_", "0")
+
+	actual := envWithoutExecutorConfig()
+
+	assert.NotContains(t, actual, "ALLEGRO_EXECUTOR_TEST_1=x")
+	assert.Contains(t, actual, "allegro_executor_TEST_2=y")
+	assert.Contains(t, actual, "TEST=z")
+	assert.Contains(t, actual, "_ALLEGRO_EXECUTOR_=0")
+
```
Comment (medzin) [inline command_test.go]: Why extra new line here?

File: command_test.go [hunk 3]
```diff
@@ -23,6 +23,21 @@ func TestIfNewCancellableCommandReturnsCommand(t *testing.T) {
 	assert.True(t, cmd.SysProcAttr.Setpgid, "should have pgid flag set to true")
 }
 
+func TestIfCombineEnvFilterExecutorConfiguration(t *testing.T) {
```
Comment (medzin) [inline command_test.go]: Add a test to check whether the process environment has remained unchanged.

File: command_test.go [hunk 4]
```diff
@@ -5,35 +5,33 @@ import (
...
-	mesos "github.com/mesos/mesos-go/api/v1/lib"
+	"github.com/mesos/mesos-go/api/v1/lib"
 	"github.com/stretchr/testify/assert"
 )
 
-func TestIfNewCancellableCommandReturnsCommand(t *testing.T) {
-	os.Environ()
-	commandInfo := newCommandInfo("./sleep 100", "ignored", false, []string{"ignored"}, map[string]string{"one": "1", "two": "2"})
+func TestIfNewCancellableCommandReturnsCommandWithoutExecutorEnv(t *testing.T) {
+	os.Setenv("ALLEGRO_EXECUTOR_TEST_1", "x")
```
Comment (medzin) [inline command_test.go]: Add defer with `os.Unsetenv` for these variables, you are modifying env for all tests here.

Comment (janisz) [reply command_test.go]: Fixed

**Pull request merged**